app/shared/design-system/components/tooltip.tsx

```diff
@@ -54,7 +59,7 @@ export const Tooltip = ({
           size="1"
           as="p"
           css={{
-            color: "$loContrast",
+            color: "currentColor",
             lineHeight: multiline ? "20px" : undefined,
```

Comment:
probably my fault, but we shouldn't use arbitrary values, use the theme

Reply:
currentColor will never be a an aribiatry value, it will use the value defined in the parent
https://css-tricks.com/cascading-svg-fill-color/

Reply:
I ment the lineHeight

Reply:
Done!